source/Calamari.Tests/KubernetesFixtures/Scripts/KubernetesDeployment.sh

```diff
@@ -227,6 +227,9 @@ function deploy_customresources {
                 done
             else
                 write_plainerror "\"kubectl apply -o json\" returned invalid JSON."
+                write_plainerror "---------------------------"
+                write_plainerror "$newCustomResources"
+                write_plainerror "---------------------------"
```

Comment:
This was a difference I found between `KubernetesDeployment.ps1` and `KubernetesDeployment.sh`. I've made it match so I didn't have adjust the test to handle different platform/script types but in the Command it behaves the same way as the Original `KubernetesDeployment.ps1` script.

Reply:
Same as for the powershell script, I don't think it's worth porting it back to server as we're phasing out the use of the scripts anyway.